README.md

```diff
@@ -223,7 +223,7 @@ Of course, Gorgonia also supports the more traditional symbolic differentiation
 	// dz/dy: 1 | 1
 ```
 
-Currently Gorgonia only performs backwards mode automatic differentiation (aka backpropagation), although one may observe the vestiges of an older version which supported forwards mode differentiation in the existence of `*dualValue`. It may return in the future.
+Currently, Gorgonia only performs backward mode automatic differentiation (aka backpropagation), although one may observe the vestiges of an older version which supported forwards mode differentiation in the existence of `*dualValue`. It may return in the future.
```

Comment:
"backwards mode" is correct. "backward mode" is wrong. Either way, the superior version would be to use "reverse mode"